doc/versions.md

```diff
@@ -0,0 +1,11 @@
+# Format versions
+| Version | Start date |
+|-|-|
+[1.0.0.0](formats/1.0.0.0.md) | 15/03/2024 |
+
+
+
+# Tools
+| Name | Version | Supported formats | Release date |
+|-|-|-|-|
+| OpenTAP.Plugins.Parquet.ParquetResultListener | 0.2.0 | 1.0.0.0 | 28/11/2023
```

Comment:
I am not sure how I should use the file. In all cases, Isn't it a bit dangerous to write the tool itself into the list? Then we know for sure it will go out of date :)

Reply:
This is here so it can be used to cross reference if tools are compatible with each other. It is not meant to be an exhaustive list, just provide a general outline of whether or not tools are compatible.

Reply:
I still dont understand. Are are you going to write: Test Automation Result viewer and PathWave Test Automation Cloud Results Viewer here as well?

Reply:
That was the idea, but it has been removed now